### PR TITLE
New Icons

### PR DESCRIPTION
**Pull request by rohats871**
Screenshot 2021-03-17 at 11 19 46 AM

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -32,8 +32,14 @@ dependencies:
...
   lint: ^1.0.0
-  path_provider: ^2.0.1
-  google_fonts: ^2.0.0-nullsafety.0
+  ionicons: ^0.1.2
+
+
+
+  # The following adds the Cupertino Icons font to your application.
+  # Use with the CupertinoIcons class for iOS style icons.
+  cupertino_icons: ^1.0.0
```
Comment (infiniteoverflow) [inline pubspec.yaml:41]: Kindly remove this line "  cupertino_icons: ^1.0.0" . This package has already been included on top.

**Review by infiniteoverflow — CHANGES_REQUESTED** [review]

File: lib/Screens/home_screen.dart [hunk 2]
```diff
@@ -1,9 +1,13 @@
 import 'package:flutter/material.dart';
-import 'package:book_donation/Services/google_sign_in.dart';
-import 'intro_screen.dart';
-import 'package:book_donation/Services/facebook_sign_in.dart';
 import 'package:google_fonts/google_fonts.dart';
+import './chat_screen.dart';
+import './profile_screen.dart';
+import './notifs_screen.dart';
```
Comment (infiniteoverflow) [inline lib/Screens/home_screen.dart:5]: Kindly dont use Relative Path , use "package:" to specify the path

**Review by infiniteoverflow — CHANGES_REQUESTED** [review]

**Comment by infiniteoverflow** [comment]
@rohats871 You havnt pulled the changes from the main branch properly. Many files are missing from your lib/screens folder. Make sure your fork is even with the main repo then work on any new issue.

File: lib/Screens/home_screen.dart (hunk 2 above)
Comment (rohats871) [reply lib/Screens/home_screen.dart:5]: im sorry but i didnt add it

**Pull request closed without merging**